### PR TITLE
[cs] improve configured rules types

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: .github/workflows/rector_ci.yaml [hunk 1]
```diff
@@ -54,4 +54,4 @@ jobs:
                     author_email: tomas@getrector.org
                 env:
                     # to get push access
-                    token: ${{ secrets.ACCESS_TOKEN }}
+                    GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```
Comment (TomasVotruba) [inline .github/workflows/rector_ci.yaml:57]: This was a bug.

@samsonasik It might help with automated pushing to the repository

Comment (samsonasik) [reply .github/workflows/rector_ci.yaml:57]: @TomasVotruba Thank you

Comment (TomasVotruba) [reply .github/workflows/rector_ci.yaml:57]: It's a rough ugess, I have not tested it nor came across the issue yet.

If that happens again, please link the pull-request and don't send any more commits. I'll try to investigate it

Comment (samsonasik) [reply .github/workflows/rector_ci.yaml:57]: @TomasVotruba  , it happen again, see latest commit at https://github.com/rectorphp/rector/pull/4489

Comment (samsonasik) [reply .github/workflows/rector_ci.yaml:57]: @TomasVotruba commit https://github.com/rectorphp/rector/pull/4489/commits/271dec62f91750d443c369808f0bd843d653625b only have 1 running action (kodiakhq).

**Pull request merged**